src/helpers/imageHelper.js

```diff
@@ -12,6 +12,11 @@ export const imageHeight = (width) => {
   return 180 * factor;
 };
 
+// TODO: image aspect ratio based on image height
+// export const imageWidtht = (height) => { const factor = height / 360;
+// return 180 * factor;
+// };
```

Comment:
is this comment still necessary?

Reply:
if you don't want this anymore, I would say no it's no necessary anymore

Reply:
we don't need it currently, so it brings no benefit to keep it